drivers/ethernet/Kconfig.sam_gmac

```diff
@@ -22,7 +22,12 @@ config ETH_SAM_GMAC_NAME
 config ETH_SAM_GMAC_QUEUES
 	int "Number of hardware TX and RX queues"
 	default 1
-	range 1 3
+	range 1 1 if SOC_SERIES_SAM4E
+	range 1 3 if !SOC_SERIES_SAM4E && \
+		     !SOC_ATMEL_SAME70_REVB && \
+		     !SOC_ATMEL_SAMV71_REVB
+	range 1 6 if SOC_ATMEL_SAME70_REVB || \
+		     SOC_ATMEL_SAMV71_REVB
```

Comment:
When the DT adds the value for the maximum number of hardware TX and RX queues supported in the future, that value should be used to specify the range.